docs/samples/python/text-analytics/index.md

```diff
@@ -0,0 +1,32 @@
+# Azure Text Analytics Integration
+
+## Introduction
+
+Azure Text Analytics is a cloud-based service that provides advanced natural
+language processing over raw text. One of its main functions includes 
+Named Entity Recognition (NER), which has the ability to identify different
+entities in text and categorize them into pre-defined classes or types.
+This document will demonstrate Presidio integration with Azure Text Analytics.
+
+## Supported entity categories in the Text Analytics API
+Text Analytics supports multiple entity categories that can be returned by NER.
+NER runs a predictive model to identify and categorize named entities from an input document.
+NER latest version includes the ability to detect personal (PII) and health (PHI) information.
```

Comment:
Is this called NER on their documentation? It is a little confusing here.

Reply:
Yes, taken from here
https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/how-tos/text-analytics-how-to-entity-linking?tabs=version-3-preview

Reply:
This sentence is confusing to me: "Text Analytics supports multiple entity categories that can be returned by NER" and "NER runs a predictive model"...

I would keep it simple because people might not know the jargon of TA. 
How about:

Text Analytics supports multiple PII entity categories. The Text Analytics service runs a predictive model to identify and categorize named entities from an input document. The service's latest version includes the ability to detect personal (PII) and health (PHI) information. A list of all supported entities can be found in the [official documentation](https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/named-entity-types?tabs=personal).

Reply:
Thanks, sounds good